SpatialGDK/Documentation/content/ability-system.md

```diff
@@ -1,10 +1,13 @@
 # Gameplay Ability System on the GDK for Unreal
 
-The [Gameplay Ability System](https://docs.unrealengine.com/en-us/Gameplay/GameplayAbilitySystem) is a highly flexible framework for building abilities and attributes of the type you might find in an RPG, MMO, or MOBA. However, as it makes use of lots of the advanced parts of the Unreal networking stack, the GDK for Unreal doesn't *currently* support it in it's entirety. On this page we'll list the workarounds that are required if you want to use the Ability System with the Alpha release of the GDK. We are working to remove some of these limitations in the future.
+The [Gameplay Ability System](https://docs.unrealengine.com/en-us/Gameplay/GameplayAbilitySystem) is a flexible framework for building abilities and attributes of the type you might find in an RPG, MMO, or MOBA. However, as it makes use of lots of the advanced parts of the Unreal networking stack, the GDK for Unreal doesn't *currently* support it in it's entirety. 
+
+This page lists the workarounds that you need to use with the Gameplay Ability System in the alpha release of the GDK. We are working to remove some of these limitations in the future.
+
+## Current Gameplay Ability System workarounds
+1. We currently do not support replicated gameplay abilities. You must ensure that any `UGameplayAbility` has its `ReplicationPolicy` set to `ReplicateNo`. Replicated gameplay abilities will be available once we support dynamic Actor components in the GDK.
+2. You must add any `UAttributeSet` objects to the `Actor` that owns the `AbilitySystemComponent`, and you must add them to the `Actor` as `DefaultSubObjects`. For example -
```

Comment:
Any reason "Actor" here is monospaced? We wouldn't usually do this: Monospace is for any names we use which are in code or file names, file paths and file extensions.

Reply:
None that I can see. I just followed the pattern from the original text. Removed monospace.